bundle/permissions/workspace_root.go

```diff
@@ -77,3 +88,29 @@ func getWorkspaceObjectPermissionLevel(bundlePermission string) (workspace.Works
 		return "", fmt.Errorf("unsupported bundle permission level %s", bundlePermission)
 	}
 }
+
+func isWorkspaceSharedRoot(path string) bool {
+	return strings.HasPrefix(path, "/Workspace/Shared/")
+}
+
+// checkWorkspaceRootPermissions checks that if permissions are set for the workspace root, and workspace root starts with /Workspace/Shared, then permissions should be set for group: users
+func checkWorkspaceRootPermissions(b *bundle.Bundle) diag.Diagnostics {
```

Comment:
Function name and comment no longer describe what it does. Please TAL before merging.